tests/foreman/api/test_contentviewversion.py

```diff
@@ -11,33 +11,170 @@
+    """Tests for content view version promotion."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create some entities for all tests."""
+        super(ContentViewVersionPromoteTestCase, cls).setUpClass()
+        cls.org = entities.Organization().create()
+        cls.lce1 = entities.LifecycleEnvironment(organization=cls.org).create()
+        cls.lce2 = entities.LifecycleEnvironment(
+            organization=cls.org,
+            prior=cls.lce1
+        ).create()
+        default_cv = entities.ContentView(
+            organization=cls.org,
+            name=u'Default Organization View'
+        ).search()
+        # There should be only 1 record returned
+        assert len(default_cv) == 1
+        # There should be only 1 version
+        assert len(default_cv[0].version) == 1
+        cls.default_cv = default_cv[0].version[0].read()
+
+    def test_positive_promote_valid_environment(self):
+        """@Test: Promote a content view version to 'next in sequence'
+        lifecycle environment.
+
+        @Assert: Promotion succeeds.
+
+        @Feature: Content View Version
+        """
+        # Create a new content view...
+        cv = entities.ContentView(organization=self.org).create()
+        # ... and promote it.
+        cv.publish()
+        # Refresh the entity
+        cv = cv.read()
+        # Check that we have a new version
+        self.assertEqual(len(cv.version), 1)
+        version = cv.version[0].read()
+        # Assert that content view version is found in 1 lifecycle
+        # environments (i.e. 'Library')
+        self.assertEqual(len(version.environment), 1)
+        # Promote it to the next 'in sequence' lifecycle environment
+        promote(version, self.lce1.id)
+        # Assert that content view version is found in 2 lifecycle
+        # environments.
+        version = version.read()
+        self.assertEqual(len(version.environment), 2)
 
-    def test_negative_promote_2(self):
-        """@Test: Promote a content view version using an invalid environment.
+    def test_positive_promote_out_of_sequence_environment(self):
+        """@Test: Promote a content view version to a lifecycle environment that
+        is 'out of sequence'.
+
+        @Assert: The promotion succeeds.
+
+        @Feature: Content View Version
+        """
+        # Create a new content view...
+        cv = entities.ContentView(organization=self.org).create()
+        # ... and publish it.
+        cv.publish()
+        # Refresh the entity
+        cv = cv.read()
+        # Check that we have a new version
+        self.assertEqual(len(cv.version), 1)
+        version = cv.version[0].read()
+        # The immediate lifecycle is lce1, not lce2
+        promote(version, self.lce2.id, force=True)
+        # Assert that content view version is found in 2 lifecycle
+        # environments.
+        version = version.read()
+        self.assertEqual(len(version.environment), 2)
+        print version.environment
```

Comment:
It seems as a mistake as we never used print statement before and basically we do not need them as we can use assertion for all scenarios